scripts/init.js

```diff
@@ -4,7 +4,12 @@ const path = require('path')
 const validateConfig = require('../lib/validateConfig')
 const { Red, Yellow, Green } = require('../lib/colors')
 
-module.exports = () => {
+const defaultOptions = {
+  skipFailure: false,
```

Comment:
We should check if initialization failed because table already exists. This is achievable with select query from INFORMATION_SCHEMA.  Alternatively if we change signature to just returning a promise we can just make catch callback () => {}.